form-val.js

```diff
@@ -10,3 +10,30 @@ function validate(e) {
     err.innerHTML = 'Email cannot be uppercase';
   }
 }
+
```

Comment:
- [ ] [OPTIONAL] You should use peer-programming to implement the changes in this PR, kindly remember this in the future projects